Source/WebKit/UIProcess/RemoteLayerTree/mac/RemoteScrollingTreeMac.mm

```diff
@@ -55,9 +55,13 @@
 
 RemoteScrollingTreeMac::~RemoteScrollingTreeMac() = default;
 
-void RemoteScrollingTreeMac::handleWheelEventPhase(ScrollingNodeID, PlatformWheelEventPhase)
+void RemoteScrollingTreeMac::handleWheelEventPhase(ScrollingNodeID nodeID, PlatformWheelEventPhase phase)
 {
-    // FIXME: Is this needed?
+    RefPtr targetNode = nodeForID(nodeID);
+    if (!is<ScrollingTreeScrollingNode>(targetNode))
+        return;
+
+    downcast<ScrollingTreeScrollingNode>(*targetNode).handleWheelEventPhase(phase);
```

Comment:
You could use `dynamicDowncast<>` to do the check and cast in one step.

Reply:
Fixed.